aiida_quantumespresso/parsers/raw_parser_pw.py

```diff
@@ -1211,8 +1243,11 @@ def parse_pw_text_output(data, xml_data={}, structure_data={}, input_dict={}, pa
             if 'site n.' in data_lines[i] and 'atom' in data_lines[i]:
                 trajectory_data['atomic_species_name'] = \
                     [data_lines[i+1+j].split()[1] for j in range(nat)]
+                # strip numbers
+                symbols = [''.join([i for i in s if not i.isdigit()]) for s in trajectory_data["atomic_species_name"]]
```

Comment:
I think we want to retain the species name and not reduce it to the elemental symbol, correct?

Reply:
I added this because, in the output `TrajectoryData` it was setting `symbols` == `atomic_species_name` (couldn't actually find in the code where this occurs) and then `trajnode.get_step_structure` failed because it could not, for example, reconcile 'Fe1' with an element. So shouldn't it be that `trajectory_data['atomic_species_name']` strores species name and trajectory_data['symbols'] stores only the symbols?

Note to self: if this is changed then `raw_parser_pw.parse_raw_output` (line 129) also needs to be changed.